apps/token-manager/app/src/script.js

```diff
@@ -200,10 +193,7 @@ function loadNewBalances(token, ...addresses) {
 
 function loadTokenSupply(token) {
   return new Promise((resolve, reject) =>
-    token
-      .totalSupply()
-      .first()
-      .subscribe(resolve, reject)
+    token.totalSupply().subscribe(resolve, reject)
   )
```

Comment:
We could probably get rid of `loadTokenSupply` entirely, since doing this is not much longer:

```js
token.totalSupply().toPromise()
```